### ISSUE TITLE
Add support for nodetool -ssl

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#2222]
*Installation details*
Scylla version (or git commit hash): 1.7

`nodetool --ssl command` for secured JMX connection.

**Comment by slivne** [comment]
@amnonh can you please check this out - I know you added support to secure jmx (and tested it somehow) - securing nodetool

**Comment by lauranovich** [comment]
Following as this is a doc issue as well.

**Comment by amnonh** [comment]
I tried to find what I did and the best I could find is:
https://groups.google.com/g/scylladb-users/c/udl1nrs6E78

Technically, scylla-jmx supports the java JMX-SSL authentication, so you can set it up like any other java SSL JMX programs and specifically Cassandra.
(scylla-jmx accept '-D' flags).

Note that by default, the remote SSL is set to false and jmx authenticate set to false.

**Comment by rmb938** [comment]
I tried to setup scylla-jmx with the ssl flags but I couldn't seem to get nodetool to use certificates for auth. Without `--ssl` it complained that it needed certificates for auth and with `--ssl` it received an EOF. 

I followed the docs here https://docs.datastax.com/en/ddacsecurity/doc/ddacsecurity/secureDDACNodetoolSSL.html to setup ` .cassandra/nodetool-ssl.properties` and it didn't seem to take.

**Comment by amnonh** [comment]
@rmb938 just to make sure, the fie is loaded from: $HOME/.cassandra/nodetool-ssl.properties not local .cassandra directory

**Comment by rmb938** [comment]
Yup I put it in `$HOME/.cassandra/nodetool-ssl.properties`. I could just be an issue on my side, maybe I did something wrong while testing. 

I can try again and report my findings.

**Comment by annastuchlik** [comment]
I think this issue is no longer valid. Anyway, I closed the corresponding documentation issue: https://github.com/scylladb/scylla-docs/issues/3151
